adr/0005-code-formatting.md

```diff
@@ -22,10 +22,10 @@ base. In place, a codebase with such formatting is:
 - We use [Black](https://github.com/python/black) as code formatter for the back-end codebase.
 - We use Black's default options.
 - We utilize a bot (such as [the black_out bot](https://github.com/Mariatta/black_out)) to ensure PRs are appropriately formatted before being accepted.
-- We recommend and document the set-up of Black, `pylint`, and `flake8` pre-commit hooks in the development environment.
+- We recommend and document the set-up of `Black`, `pylint`, and `flake8` in the development environment.
 
 ## Consequences
 
-- We retain Flake8, but set `max-line-length = 88` and `ignore = E501, W503` to `setup.cfg`.
-- We retain pylint, but disable all formatting checks.
+- We retain Flake8, but set `max-line-length = 88` and `ignore = E501, W503, E203` to `setup.cfg`.
+- We retain pylint, but disable `bad-continuation`.
```

Comment:
We should disable all formatting check with pylint, instead, check those with Black itself.
To captains on a ship is a bad idea when it comes to formatting.

Reply:
Have you checked what rules will be disabled?

Reply:
Yes, I did.

https://github.com/PyCQA/pylint/blob/master/pylint/checkers/format.py#L127

Reply:
Thanks! Looks good.

Reply:
I run black with these settings on Hass.io and there is no problem or reason to disable all formattings

Reply:
I would say that the reason is too many cooks in the kitchen. Since the user is unable to control formatting, we shouldn't lint for it either.